packages/clay-tooltip/stories/index.tsx

```diff
@@ -44,45 +44,43 @@ storiesOf('Components|ClayTooltip', module)
 			<button title="Default">{'No Tooltip'}</button>
```

Comment:
This now shows a tooltip in the Story since it is global.

![Screen Shot 2021-04-26 at 3 37 04 PM](https://user-images.githubusercontent.com/6843530/116159570-53c27680-a6a5-11eb-8d06-c0e08890f856.png)

Reply:
Yeah, this is the limiter or regression that we will have using this implementation, we can try to reverse this with some strategies.